ajax/libs/jquery-nearest/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "jquery-nearest",
+    "description": "Find the elements in a page that are closest to a particular point or element, based on pixel dimensions",
+    "filename": "jquery.nearest.min.js",
+    "keywords": [
+        "jquery-plugin", 
+        "ecosystem:jquery", 
+        "nearest", "furthest", 
+        "touching", 
+        "pixel", 
+        "distance"
+    ],
+    "author": "Gilmore Davidson",
+    "homepage": "https://gilmoreorless.github.io/jquery-nearest/",
+    "repository": {
+        "type": "git",      
+        "url": "git+https://github.com/gilmoreorless/jquery-nearest.git"
+    },    
+    "autoupdate": {
+        "source": "git",
+        "target": "git://github.com/gilmoreorless/jquery-nearest.git",
+        "fileMap": [
+          {
+            "basePath": "src",
+            "files": [
+              "**/jquery.nearest*"
```

Comment:
Let's use jquery.nearest*.+(css|map|js) instead because it's not a good idea grab all files under `src` recursively.